conductr_cli/sandbox_main.py

```diff
@@ -62,6 +62,45 @@ def build_parser():
                             help='Set additional environment variables for each ConductR container\n'
                                  'Defaults to []',
                             metavar='')
+    run_parser.add_argument('--env-core',
+                            dest='envs_core',
+                            action='append',
+                            default=[],
+                            help='Set additional environment variables for each ConductR core process. Defaults to [].',
+                            metavar='')
+    run_parser.add_argument('--env-agent',
+                            dest='envs_agent',
+                            action='append',
+                            default=[],
+                            help='Set additional environment variables for each ConductR agent process. Defaults to [].',
```

Comment:
`Defaults to` is always on a separate line. Also, we don't use dots at the end of the line.

So it should be changed to:

```
Set additional environment variables for each ConductR agent process\n
Defaults to []
```